libs/orders/src/lib/components/order-feedback/order-feedback.tsx

```diff
@@ -52,12 +52,11 @@ export const OrderFeedback = ({ transaction, order }: OrderFeedbackProps) => {
                 : 'text-vega-red-dark dark:text-vega-red'
             }
           >
-            {`${
-              order.side === Side.SIDE_BUY ? '+' : '-'
-            } ${addDecimalsFormatNumber(
-              order.size,
-              order.market?.positionDecimalPlaces ?? 0
-            )}
+            {`${order.side === Side.SIDE_BUY ? '+' : '-'
```

Comment:
nitpick (non-blocking): seeing this pop up in a few places over the last little while. May be worth a util.